src/ProjectTemplates/test/Templates.Tests/template-baselines.json

```diff
@@ -660,8 +748,8 @@
       "AuthOption": "None"
     },
     "EmptyUseWebAssembly": {
-      "Template": "web",
-      "Arguments": "new blazor --use-server false --use-wasm --empty",
+      "Template": "blazor",
```

Comment:
So this was a bug we didn't notice in the past then?

Reply:
It was a weird inconsistency but didn't impact the test actually working correctly. I think the `Template` string is only used for reporting in logs, and doesn't affect whether the test passes or fails.